src/Makefile.am

```diff
@@ -1414,6 +1414,17 @@ endif
 nodist_EXTRA_fuzz_predefpcap_aware_SOURCES = force-cxx-linking.cxx
 endif
 
+fuzz_decodebase64_SOURCES = tests/fuzz/fuzz_decodebase64.c
+fuzz_decodebase64_LDFLAGS = $(LDFLAGS_FUZZ)
+fuzz_decodebase64_LDADD = $(LDADD_FUZZ)
+if HAS_FUZZLDFLAGS
```

Comment:
You could pick the fix of https://github.com/OISF/suricata/pull/11036/commits/964a38493f07696fd889f4907ee0dc2ff20e1d4b for this target already ;-)